cookbooks/travis_internal_base/files/default/create-users

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# vim:filetype=sh
+
+main() {
+  : "${GITHUB_USERS_ENV:=/etc/default/github-users}"
```

Comment:
is there anything that is currently creating this file? or is that to be added still?